### DISCUSSION TITLE
Search/Replace delimiters (\n \t etc..)

### DISCUSSION THREAD

**Discussion by akmalkadi** [linuxdeepin/developer-center#2952] [Features Request & Ideas | 特性请求 & 头脑风暴]
Greetings!

I am a Linux mint user and right now I am trying Deepin. I use xed (text editor) in Linux mint and I usually use search and replace. Searching for delimiters is a common practice. However, I found that searching for them in Deepin text editor is not supported yet.

delimiters: \n \t etc...

Please support searching and replacing delimiters.

Best,

**Comment by z966768** [comment]
This feature has not been added yet, any progress will be back to you here

**Comment by guoshaoyu** [comment]
Will be added as new requirements in subsequent releases.

**Comment by meiyixiang** [comment]
This is a good suggestion, I have fed back to the product, this function will be supported before June 30, 2022.
